src/controllers/admin/office.ts

```diff
@@ -42,3 +44,155 @@ export const AdminLoginHandler = async (req: Request, res: Response) => {
     });
   }
 }
+
+export const AdminDashboardHandler = async (req: Request, res: Response) => {
```

Comment:
Way too many DB calls. Merge queries and reduce it ?

Reply:
+1

Reply:
@Nandgopal-R while the current PR shows improvement and has effectively reduced the number of DB calls. I feel the queries can still be optimized. Some of them can be turned into `aggregation` queries so that we do not end returning huge amounts of data. 

Will pull down your branch and try to roll-out a fix. Then we can review it together.

Reply:
ok